core/schedule/include/schedule/schedule.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include <schedule/schedule_day.h>
+#include <database/database.h>
+
+#include <string>
+
+namespace tabetai2::core::schedule {
+
+
+class Schedule {
+public:
+    Schedule(database::Id id, std::string start_date);
+
+    database::Id id() const;
+
+    void add_day(const ScheduleDay& day);
+
+    std::string start_date() const;
```

Comment:
Do you really want to force copying of these return values each time? You could return a `const &` and let the caller decide if they need a copy or not?